Source/WebKit/DerivedSources-input.xcfilelist

```diff
@@ -687,3 +687,4 @@ $(WEBCORE_PRIVATE_HEADERS_DIR)/ProtectionSpaceBase.serialization.in
 $(WEBCORE_PRIVATE_HEADERS_DIR)/ScrollTypes.serialization.in
 $(WEBCORE_PRIVATE_HEADERS_DIR)/WebGPU.serialization.in
 $(WEBCORE_PRIVATE_HEADERS_DIR)/generate-bindings.pl
+/Applications/Xcode3.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS18.0.Internal.sdk/usr/local/include/WebKitAdditions/TextStyleManager.swift.in
```

Comment:
Oops, the latest push didn't remove this

Reply:
I deleted this.... I don't know how it came back... or how it got there in the first place...